.ci/onnx/jenkins/base_image_builder/Jenkinsfile

```diff
@@ -0,0 +1,93 @@
+// Set LABEL variable if empty or not declared
+try{ if(LABEL.trim() == "") {throw new Exception();} }catch(Exception e){LABEL="onnx && ci"}; echo "${LABEL}"
+try{ if(DOCKER_REGISTRY.trim() == "") {throw new Exception();} }catch(Exception e){DOCKER_REGISTRY="amr-registry.caas.intel.com"}; echo "${DOCKER_REGISTRY}"
```

Comment:
I don't think we should be exposing internal domain names in a public configuration file.

Reply:
Addressed